### DISCUSSION TITLE
FR: Stacking Support

### DISCUSSION THREAD

**Discussion by debkanchan** [trunk-io/docs#271] [Feature Requests]
graphite.dev provides a good support for stacking but it does a bit too much. I don't want a whole separate interface from github, I want to use github AND do stacking easily.

I feel trunk is positioned perfectly to support stacking with the launch of batching in merge queue and the existing merge graph.

**Comment by pv72895** [comment]
Hey DK!

Thank you for suggesting stacking! This feature is on the merge team's roadmap, so it will certainly make its way into merge at some point, but just not in the immediate term. Once we have a timeline, I will let you know!

In the meantime, would love to know more about how stacking fits into your workflow and how you imagine it working in merge to make sure that what we build for merge would meet your needs and others in the community. At a minimum, we'd of course expect merge to be able to submit and merge stacks as a whole, either across multiple branches (how graphite does it) or as commits within a single branch

**Comment by Ryang20718** [comment]
> merge stacks as a whole

+1 to this!

**Comment by Ryang20718** [comment]
+1 to this a year later :)

**Comment by debkanchan** [comment]
I still need this.

**Comment by EliSchleifer** [comment]
We are in the final stages of getting stacked support out there. Right now limitations with running parallel mode alongside GitHub's limited API support for merging stacks is holding us up. We will update here as we have more info
